src/System.Runtime/tests/System/Runtime/ExceptionServices/HandleProcessCorruptedStateExceptions.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime;
+using System.Runtime.ExceptionServices;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using Xunit;
+
+namespace System.Runtime.ExceptionServices.Tests
+{
+    public class HandleProcessCorruptedStateExceptionsTests : RemoteExecutorTestBase
+    {
+        [DllImport("kernel32.dll")]
+        static extern void RaiseException(uint dwExceptionCode, uint dwExceptionFlags, uint nNumberOfArguments, IntPtr lpArguments);
```

Comment:
What about Unix?  And is this allowed for UWP or will this need to be factored out for that, too?